packages/bpk-docs/src/routes/Routes.js

```diff
@@ -295,59 +323,103 @@ const Routes = (
 
       <Route path={ROUTES.NATIVE_COMPONENTS}>
```

Comment:
The `oldSite` check could only be done at this level instead of every route below